.github/workflows/linux.yml

```diff
@@ -121,6 +121,8 @@ jobs:
         env:
           CC: clang-18
           CXX: clang++-18
+          # Test fallback SHA implementations
+          CMAKE_FLAGS: '-DENABLE_OPENSSL=0'
```

Comment:
you *could* also apply this to the gcc_7 workflow since it doesn't have the correct openssl version anywy

Reply:
Oki doki, done.